packages/react-test-renderer/src/ReactShallowRenderer.js

```diff
@@ -308,6 +318,54 @@ class ReactShallowRenderer {
       );
     };
 
+    const useGenericEffect = (
+      create: () => (() => void) | void,
+      inputs: Array<mixed> | void | null,
+      isLayoutEffect: boolean,
+    ) => {
+      this._validateCurrentlyRenderingComponent();
+      this._createWorkInProgressHook();
+
+      if (this._workInProgressHook !== null) {
+        if (this._workInProgressHook.memoizedState == null) {
+          this._workInProgressHook.memoizedState = {
+            isEffectHook,
+            isLayoutEffect,
+            create,
+            inputs,
+            destroy: null,
+            run: true,
+          };
+        } else {
+          const {memoizedState} = this._workInProgressHook;
```

Comment:
IMO, this is style, but I prefer the spaces around the brackets, ex: `const { memoizedState }`

Reply:
react uses prettier right? it should make these decisions for us

Reply:
Yep, I see it now. It's set this way. Sorry!